src/main/java/org/dependencytrack/resources/v1/ProjectResource.java

```diff
@@ -141,6 +142,38 @@ public Response getProject(
         }
     }
 
+    @GET
+    @Path("/latestUpdated/{name}")
```

Comment:
The path and `ApiOperation` description should not include "updated", because a project can be updated in many different ways that do not involve uploading a BOM. Since this endpoint returns a project based on the latest BOM upload timestamp, both the path and the operations description should reflect that.

Reply:
What name would you have ? Would **latestImportedBom** suit better ? I'm happy to use any wording proposition.